src/cdk/dialog/dialog-container.ts

```diff
@@ -95,7 +96,7 @@ export class CdkDialogContainer<C extends DialogConfig = DialogConfig>
    * where there are two or more titles in the DOM at a time and the first one is destroyed while
    * the rest are present.
    */
-  _ariaLabelledByQueue: string[] = [];
+  _ariaLabelledByQueue = signal([] as string[]);
```

Comment:
Hmm, I'm not sure if we're supposed to be using signals yet.

Reply:
Okay, just let me know. I can change it but it's a whole lot easier to use a signal than the alternatives.

Reply:
In the interest of not waiting for the team to be back so we can discuss, I'd rather we skip the signal for now. We'll make a pass over the whole library to convert things to signals eventually. For now I'm thinking that we should add a method like this to the dialog container:

```
_addAriaLabelledBy(id: string) {
  this._ariaLabelledByQueue.push(id);
  this._changeDetectorRef.markForCheck();
}
```

Then the title directive can call `_addAriaLabelledBy`.

Reply:
Done. I did also keep `_ariaLabelledByQueue` public to avoid needing to update the legacy components internally.